site/content/en/docs/handbook/registry.md

```diff
@@ -44,9 +44,9 @@ One nifty hack is to allow the kubelet running in minikube to talk to registries
 with TLS certificates. Because the default service cluster IP is known to be available at 10.0.0.1, users can pull images from registries
 deployed inside the cluster by creating the cluster with `minikube start --insecure-registry "10.0.0.0/24"`.
 
-### docker on macOS
+### docker on Linux/macOS
```

Comment:
this doesn't seem right, 

docker on Linux is one way
docker on mac os and windows is the other way.

this is due to docker limitaitons on non-linux

and the limitation is, on non-linux we can not use the port 5000. but use a different random port.

Reply:
this doc needs to be reviewed from scratch to make sure the language is correct all the way.